core/sql/regress/seabase/EXPECTED010

```diff
@@ -6113,6 +6114,7 @@ DESCRIPTION
   begin_keys(incl) ....... 4,b,-2147483648
   end_keys(excl) ......... 4,b,2
   new_rec_expr ........... (D assign 'upd')
+  predicate .............. (C < 2)
```

Comment:
It looks like we have added an exe pred that would clearly be covered by the begin/end key. I was expecting the changes I saw to have the opposite effect (i.e. miss adding necessary exe preds), so clearly my understanding is incomplete. However this is not that desirable also. Any ideas why this happened?